### PR TITLE
Update unit as  inference/sec

### PR DESCRIPTION
**Pull request by pramenku**
As a user, I am trying to enable the unit self-explanatory.  

Usually, we see unit format as  <some value>  <what the value for>/<time>. Example: X images/sec,  X frames/sec or etc where images or frames helping user to understand , what the value for.

Below Rate unit, until, I read doc or check with owner, I am not sure what it means.  Just for a layman, if we keep like below, they might not get it. Rate: X/sec

But, if we keep, X inference/sec, it will help clearly to know what this value for.  The Rate is  how many inferences of the given inputs can be processed per second so X inference/sec will help user for the same.

For advance user, it /sec might be ok but I am trying to cover all.

### CODE REVIEW

File: src/program.cpp [hunk 1]
```diff
@@ -936,7 +936,7 @@ void program::perf_report(std::ostream& os,
     os << std::endl;
 
     os << "Batch size: " << batch << std::endl;
-    os << "Rate: " << rate * batch << "/sec" << std::endl;
+    os << "Rate: " << rate * batch << "inference/sec" << std::endl;
```
Comment (umangyadav) [inline src/program.cpp:939]:
```suggestion
    os << "Rate: " << rate * batch << "inferences/sec" << std::endl;
```

Comment (umangyadav) [reply src/program.cpp:939]: Minor comment

**Comment by pramenku** [comment]
Hi @causten @umangyadav 
Please feel free to merge as I don't have the permission.

After that need to put the same in master and later 6.0 release branch

**Review by pfultz2 — APPROVED** [review]

**Comment by kahmed10** [comment]
This formatting looks wrong. Before it was /sec, which is fine to not have a space between the number. But now we need to modify it as such:
```
    os << "Rate: " << rate * batch << " inferences/sec" << std::endl;
```

**Comment by causten** [comment]
> This formatting looks wrong. Before it was /sec, which is fine to not have a space between the number. But now we need to modify it as such:
> 
> ```
>     os << "Rate: " << rate * batch << " inferences/sec" << std::endl;
> ```

I agree...

```
Batch size: 16
Rate: 1337.84inferences/sec
Total time: 11.9596ms
Total instructions time: 12.9021ms
Overhead time: 0.0595123ms, -0.942473ms
Overhead: 0%, -8%
```

Honestly it would parse a lot easier if it looked like...

```
Batch size: 16
Rate (inf/sec): 1337.84
Total time (ms): 11.9596
Total instructions time (ms): 12.9021
Overhead time (ms): 0.0595123, -0.942473
Overhead: 0%, -8%
```

**Comment by umangyadav** [comment]
> Honestly it would parse a lot easier if it looked like... 
Batch size: 16
Rate (inf/sec): 1337.84
Total time (ms): 11.9596
Total instructions time (ms): 12.9021
Overhead time (ms): 0.0595123, -0.942473
Overhead: 0%, -8% 

I like that better. also i'll need to change my script to reflect this change now. I'll do that now. 

https://github.com/ROCmSoftwarePlatform/YModel-Scripts/blob/90bc6e1dc1b600bc062cd3d55c0791ec6387c334/run_ymodel.sh#L147

**Comment by pramenku** [comment]
Rate (inf/sec): 1337.84
>> This also looks fine. Can we get this merged in develop, master and release 6.0 branch.

After that we need to also modify the our automation code.

cc: @umangyadav @causten

**Pull request merged**